src/Pages/Notes/Edit.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { useState } from "react"
+import { useNote } from "../../Context/NoteContext"
+import { Header } from "../HomePage/Header"
+import { Sidebar } from "../HomePage/Sidebar/Sidebar"
+import axios from 'axios'
+import { useEffect } from 'react'
+import './Note.css'
+// import { SingleNote } from "./SingleNote"
+export const EditNote = () =>{
+    const {note, addNote, updateNote} = useNote()
+    const [newNote, setNewNote] = useState(note)
+    const [open , setOpen] = useState(false)
+
+    const handleNotes = (e) =>{
+       setNewNote({...newNote, [e.target.name] : e.target.value})
+    }
+    console.log("newnote:",newNote)
+    console.log("from context:",note)
+    const addNotes = () =>{
+        const {title} = newNote
+        console.log(newNote)
+        // addNote(newNote)
+        setNewNote({
+            content:'',
+            "title":newNote.title,
```

Comment:
why is title in string?

Reply:
that's not part of this pr I was just trying to do edit functionality